### ISSUE TITLE
os._exit is not private

### ISSUE THREAD

**Issue by sersorrel** [microsoft/pyright#2769]
**Describe the bug**
Importing `os._exit` in strict mode results in the error `"_exit" is private and used outside of the module in which it is declared`.

**To Reproduce**
```python
from os import _exit
```

**Expected behavior**
pyright should not error out when you use the correct, [documented](https://docs.python.org/3/library/os.html#os._exit), function to exit after a fork.

**Screenshots or Code**
```python
# pyright: strict
import os
os._exit(0)
```

**VS Code extension or command-line**
command-line tool, version 1.1.187

**Comment by erictraut** [comment]
The name indicates otherwise, since it begins with an underscore. Python has no keyword for marking attributes or methods as private, so the underscore convention is the only thing a type checker has to go on. We generally don't special-case issues like this in the type checking logic. The workaround is to disable the optional `reportPrivateUsage` check in that file or for the project if you don't want to see this error.
